Zend/zend_builtin_functions.stub.php

```diff
@@ -107,13 +105,11 @@ function trigger_error(string $message, int $error_level = E_USER_NOTICE): true
 /** @alias trigger_error */
 function user_error(string $message, int $error_level = E_USER_NOTICE): true {}
 
-/** @return callable|null */
-function set_error_handler(?callable $callback, int $error_levels = E_ALL) {}
+function set_error_handler(?callable $callback, int $error_levels = E_ALL): ?callable {}
```

Comment:
The reason these return types are not documented is described in Nikita's comment: https://github.com/php/php-src/pull/5618#pullrequestreview-417387828